.github/ISSUE_TEMPLATE/build_system.yml

```
@@ -0,0 +1,80 @@
+name: Build System
+description: Report an issue with the OS build system, NOT with the OS itself
+
+body:
+  - type: markdown
+    attributes:
+      value: |
+        This repository is for the elementary OS build system and **not a catch-all for the OS itself**. Please do not file issues for the operating system itself or apps here, as they will be closed or need to be manually transferred.
+
+        If you are on elementary OS and trying to report an issue with something in the OS itself:
+
+        1. Open System Settigns → About → Report a Problem
+        2. Choose the appropriate category and component
+        3. Follow the prompts to report a problem
+
+        If you are not on elementary OS, please:
+
+        1. Visit https://github.com/elementary
+        2. Use your best effort to find the correct repository
+        3. Check for similar or duplicate issues to what you're trying to report
+        4. If none are found, file a new issue against that repository
+
+        Please remember, the more time that we have to spend triaging issues is less time we have to address the ones that are already open. :)
+
+  - type: textarea
+    attributes:
+      label: What Happened?
+      description: A detailed description of unexpected behavior will help us address your issue more quickly. Do your best!
+    validations:
+      required: true
+
+  - type: textarea
+    attributes:
+      label: Steps to Reproduce
+      description: Explain the exact steps one would take to experience the issue. If applicable, add screenshots or screen recordings.
+      placeholder: |
+          1. Went here 
+          2. Did this 
+          3. Clicked that 
+    validations:
+      required: true
+
+  - type: textarea
+    attributes:
+      label: Expected Behavior
+      description: Explain how what happened is different from what you wanted to happen.
+    validations:
+      required: true
+
+  - type: dropdown
+    attributes:
+      label: OS Version
+      description: The version of elementary OS you are using, or other operating system
+      options:
+        - 6.x (Stable, from our homepage)
+        - 6.x (Early Access)
+        - 7.x (Early Access)
+    validations:
+      required: true
+
+  - type: dropdown
+    attributes:
+      label: Hardware Platform
+      description: The processor architechture of the image you're installing
```

```suggestion
      description: The processor architecture of the image you're installing
```

Typo, but, would this be better as something like the variant of the image you're building/using?

If we're saying this is only for reporting problems with the build scripts, then people shouldn't be reporting issues here if they're installing the image?